packages/frontend/components/Image.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+
+export default function Image({ image }) {
```

Comment:
Hi, this is a very good component!
It would be great if you can add `type` for the image parameter
Just to make sure it's consistent across the components

Reply:
fixed